m/main_glider_data_processing_dt.m

```diff
@@ -233,6 +252,7 @@
   netcdf_l0_file = strfstruct(config.paths_local.netcdf_l0, deployment);
   netcdf_l1_file = strfstruct(config.paths_local.netcdf_l1, deployment);
   netcdf_l2_file = strfstruct(config.paths_local.netcdf_l2, deployment);
+  matlab_l2_file = strfstruct(config.paths_local.matlab_l2, deployment);
```

Comment:
Are you modifying the files in the Github interface and in your local machine at the same time?
If yes, you will end up with conflicts. The easier way is to make all the changes
in your local machine and commit them, and then push to the github repo.
If you make any change via the github interface, you should `git pull` it as soon as possible,
to prevent conflicts later on.